ui/src/router/index.js

```diff
@@ -31,7 +31,7 @@ const router = new Router({
 });
 
 router.beforeEach((to, from, next) => {
-  const isAuthenticated = store.getters.isAuthenticated;
+  var isAuthenticated = store.getters.isAuthenticated;
```

Comment:
The `var` should remain a constant: its value shouldn't be changed, only retrieved from the store.

Reply:
Sorry for the late response.

Reply:
Done 👍🏻